src/utils/format.ts

```diff
@@ -300,3 +300,16 @@ export function formatExecutedPriceToDisplay(
 
   return `${formattedPrice} ${baseSymbol}`
 }
+
+/**
+ * @param amount BigNumber integer amount
+ * @param token Erc20 token
+ */
+export function defaultAmountFormatPrecision(amount: BigNumber, token?: TokenErc20 | null): string {
+  return formatSmart({
+    amount: amount.toString(10),
+    precision: token?.decimals || 0,
+    decimals: HIGH_PRECISION_DECIMALS,
```

Comment:
For this you can either not include `decimals` to use the default (https://github.com/gnosis/dex-js/blob/c8fa22f01ab1e04552fa02b55c89ccaa94f62a4f/src/const.ts#L26) or to be explicit create a const like `MEDIUM_PRECISION_DECIMALS` or `DEFAULT_PRECISION_DECIMALS` and set it to `4`.

`8` feels too much on this view.

Only thing to keep in mind is to update the value in the input for `smallLimit` below